utils/debug/PointCloudDebug.js

```diff
@@ -21,41 +21,53 @@ function showController(gui, name) {
     getController(gui, name).__li.style.display = '';
 }
 
-function setupControllerVisibily(gui, mode) {
-    if ([PNTS_MODE.INTENSITY, PNTS_MODE.ELEVATION, PNTS_MODE.SCAN_ANGLE].includes(parseInt(mode, 10))) {
+function setupControllerVisibily(gui, displayMode, sizeMode) {
+    displayMode =  parseInt(displayMode, 10);
+    if ([PNTS_MODE.INTENSITY, PNTS_MODE.ELEVATION, PNTS_MODE.SCAN_ANGLE].includes(displayMode)) {
         showController(gui, 'gradient');
     } else {
         hideController(gui, 'gradient');
     }
-    if (PNTS_MODE.INTENSITY === parseInt(mode, 10)) {
+    if (PNTS_MODE.INTENSITY === displayMode) {
         showController(gui, 'minIntensityRange');
         showController(gui, 'maxIntensityRange');
     } else {
         hideController(gui, 'minIntensityRange');
         hideController(gui, 'maxIntensityRange');
     }
-    if (PNTS_MODE.ELEVATION === parseInt(mode, 10)) {
+    if (PNTS_MODE.ELEVATION === displayMode) {
         showController(gui, 'minElevationRange');
         showController(gui, 'maxElevationRange');
     } else {
         hideController(gui, 'minElevationRange');
         hideController(gui, 'maxElevationRange');
     }
-    if (PNTS_MODE.SCAN_ANGLE === parseInt(mode, 10)) {
+    if (PNTS_MODE.SCAN_ANGLE === displayMode) {
         showController(gui, 'minAngleRange');
         showController(gui, 'maxAngleRange');
     } else {
         hideController(gui, 'minAngleRange');
         hideController(gui, 'maxAngleRange');
     }
+
+    sizeMode =  parseInt(sizeMode, 10);
+    if (sizeMode === PNTS_SIZE_MODE.VALUE) {
+        hideController(gui, 'minAttenuatedSize');
+        hideController(gui, 'maxAttenuatedSize');
+        showController(gui, 'pointSize');
+    } else {
+        hideController(gui, 'pointSize');
```

Comment:
I think we should not hide `pointSize` parameter as it used in the attenuation calculation of `gl_Pointsize` in the shader.